src/App.js

```diff
@@ -22,6 +23,7 @@ function App() {
         <Route path="/topics" element={<TopicList />}/>
         <Route path="/users" element={<UserList/>}/>
         <Route path="/articles?query" element={<Articles />}/>
```

Comment:
You shouldn't need a separate route for the query - it's all a part of /articles